README.md

```diff
@@ -354,9 +346,10 @@ nRF Connect respects the `main` field in the app's package.json, so the entry fi
 
 When implementing the `map*Dispatch` methods, apps can create and dispatch custom actions. When dispatching actions, the action creator get access to an `api` object that enables using the logger, serial ports, BLE, and programming. Below is an example that adds a log message when the user selects a serial port.
```

Comment:
This paragraph is out of date now. I think we might need to rewrite this whole section. Something like:

#### Included libraries/APIs

The nRF Connect framework includes a set of libraries/APIs that apps can import. See below for a full list.

(table here)

##### Example

Below is an example that imports the logger instance, and adds a log message when the user selects a serial port.

```
import { logger } from 'nrfconnect/core';

module.exports = {
    mapSerialPortSelectorDispatch: (dispatch, props) => ({
        ...props,
        onSelect: port => logger.info(`Serial port ${port.comName} was selected`),
    }),
};
```